qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java

```diff
@@ -28,6 +29,9 @@
 public class UserAccountSteps extends Steps {
 
 	private static final String SYSTEM_ALERT = "Atleast 8 character(s) are required";
+	private static final String CLERK = RandomStringUtils.randomAlphabetic(8);
+	private static final String NURSE = RandomStringUtils.randomAlphabetic(8);
+	private static final String DOCTOR = RandomStringUtils.randomAlphabetic(8);
```

Comment:
Ohh cool i can see its being used [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/AddVisitTypeTest.java#L31). perhaps it could be a great refactor.